docs/getting-started/install-guide.md

```diff
@@ -29,22 +37,24 @@ The Windows Installer (`.msi` file) is located here: [https://github.com/decred/
 
 `dcrinstall` is an automatic installer and upgrader for the Decred software. The newest release can be found here: [https://github.com/decred/decred-release/releases](https://github.com/decred/decred-release/releases). Binaries are provided for Windows, OSX/macOS, Linux, OpenBSD, and FreeBSD. Executing installer will install `dcrd`, `dcrwallet`, and `dcrctl`. Instructions are provided for Mac, Linux, and Windows below (assumed proficiency for *BSD users).
 
+This method is recommended over the traditional manual installation. `dcrinstall` will automatically download the precompiled, signed binaries found on Github, verify the signatures of the binaries, copy them to a specific folder dependant on OS, create configuration files with settings to allow the 3 applications to communicate with each other, and run you through the wallet creation process. After running through `dcrinstall`, you will be able to launch the software with no additional configuration.
+
 > OSX/macOS:
 
 1. Download the correct file:
 
     For 32-bit computers, download the `dcrinstall-darwin-386-v0.8.2` file. <br />
     For 64-bit computers, download the `dcrinstall-darwin-amd64-v0.8.2` file.
 
-2. Make dcrinstall-darwin-xxxx-vx.x.x an executable within your terminal:
+2. Make dcrinstall-darwin-xxxx-vx.x.x an executable within your terminal, and run it:
 
     Navigate to the directory where the dcrinstall file was downloaded using the `cd` command, run chmod with u+x mode on the dcrinstall file, and run the executable that is created. Below is an example of the commands (change directories or filename as needed):
     
     `cd ~/Downloads/` <br />
     `chmod u+x dcrinstall-darwin-amd64-v0.8.2` <br />
     `./dcrinstall-darwin-amd64-v0.8.2`
     
-3. The binaries for `dcrd`, `dcrwallet`, and `dcrctl` can then be found in the `~/decred/` directory.
+3. The executable binaries for `dcrd`, `dcrwallet`, and `dcrctl` can then be found in the `~/decred/` directory. Before the `dcrinstall` completes, you will be taken to the wallet creation prompt. Follow the steps within the [Wallet Creation Walkthrough](/getting-started/user-guides/dcrwallet-setup.md#wallet-creation-walkthrough) of the dcrwallet Setup guide to finish.
```

Comment:
That kind of reads weird.  Probably should either be: 'Before `dcrinstall` completes' or Before the `dcrinstall` process completes'